### PR TITLE
Adds manage script to package application in containers

### PR DESCRIPTION
**Pull request by amanji**
Signed-off-by: Akiff Manji <akiff.manji@gmail.com>

### CODE REVIEW

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
You'll need to add a gitattributes file to ensure the line endings of scripts and executable files are set properly.  Example; https://github.com/bcgov/aries-vcr/blob/master/.gitattributes

Also ensure scripts and executables are marked as executable and marked as such in Git.

As it stands, this will not build/run on a Windows platform.

**Comment by WadeBarnes** [comment]
The application is connecting (hard coded) to `http://localhost:8081` rather then connecting to the locally running instance of `aries-vcr` as configured by the `manage` script and docker configuration.

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -16,9 +16,13 @@ services:
...
     networks:
+      - web
       - vcr
     ports:
       - "${WEB_HTTP_PORT}:8080"
 
 networks:
-  vcr:
\ No newline at end of file
+  web:
```
Comment (esune) [inline docker/docker-compose.yml:25]: Do we even need to declare a network, having only one component in this compose file?

Comment (amanji) [reply docker/docker-compose.yml:25]: We should at least reference the external network

Comment (amanji) [reply docker/docker-compose.yml]: Removed the default network, but still referencing `vcr` externally

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
The application is still making explicit calls to the `aries-vcr` API via the URL configured in `.../aries-vcr-client/src/environments/environment.ts`; `http://localhost:8081/api` for example.  Rather than making relative API calls to it's own host address.

The API calls need to be relative to allow the application's host address to change dynamically.

Discussed directly with @amanji

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**